Looks like it runs into a different issue after this:
```
checking for Qt5Core >= 5.9.5... no
configure: error: Qt5Core >= 5.9.5 not found
This file contains any messages produced by compilers while
running configure, to aid debugging if configure makes a mistake.
```

--- a/.cirrus.yml
+++ b/.cirrus.yml
@@ -187,8 +187,9 @@ task:
 
 task:
   name: 'macOS 11 native [gui] [no depends]'
-  macos_brew_addon_script:
-    - brew install boost libevent berkeley-db4 qt miniupnpc libnatpmp ccache zeromq qrencode sqlite libtool automake pkg-config gnu-getopt
+  brew_install_script:
+    - brew update
+    - brew install boost libevent berkeley-db4 qt@5 miniupnpc libnatpmp ccache zeromq qrencode sqlite libtool automake pkg-config gnu-getopt
   << : *GLOBAL_TASK_TEMPLATE
   osx_instance:
     # Use latest image, but hardcode version to avoid silent upgrades (and breaks)

